### ISSUE TITLE
Add `focus-within` to menus with `hover` pseudo classes

### ISSUE THREAD

**Issue by orkhan-huseyn** [asan-developers/government-website-template#2]
Dropdown menus should be accessible with keyboard. Right now they only open when `hover`in over them with mouse. Add `:focus-within` pseudo class to enable same functionality to keyboard users.
